src/v/cloud_storage_clients/abs_client.cc

```diff
@@ -610,7 +612,8 @@ ss::future<abs_client::list_bucket_result> abs_client::do_list_objects(
   std::optional<object_key> start_after,
   std::optional<size_t> max_keys,
   [[maybe_unused]] std::optional<ss::sstring> continuation_token,
-  ss::lowres_clock::duration timeout) {
+  ss::lowres_clock::duration timeout,
+  std::optional<item_filter>) {
```

Comment:
This needs implementing, right?  I think we have the same problem with ABS bucket listings that we do in S3.

Reply:
Added parser implementation for ABS